### PR TITLE
Code Review Request 

### PR DESCRIPTION
**Pull request by nyanyehtun-simon**
check formatting, naming and logic for Practical 5 codes
@Reginald96

### CODE REVIEW

File: Prac_05/color_names.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
+COLORS = {
```
Comment (Reginald96) [inline Prac_05/color_names.py:1]: very good coding

File: Prac_05/count_words.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
+user_input_text = ''
```
Comment (Reginald96) [inline Prac_05/count_words.py:1]: clean and easy to understand

**Review by Reginald96 — COMMENTED** [review]
reviewed codes

File: Prac_05/color_names.py (hunk 1 above)
Comment (nyanyehtun-simon) [reply Prac_05/color_names.py:1]: Thanks

**Comment by nyanyehtun-simon** [comment]
Rebase and merge code from Feedback branch to master branch

**Pull request merged**